README.md

```diff
@@ -886,25 +889,122 @@ A backend represents a way to run the user's command specified in the `task` sec
 
-* The Cromwell process can write to the current working directory
 * (`local` backend) Subprocesses that Cromwell launches can use child directories that Cromwell creates as their CWD.  The subprocess must have write access to the directory that Cromwell assigns as its current working directory.
-* (`sge` backend) Jobs launched with `qsub` can use child directories as the working directory of the job, and write files to those directories.
+* (`sge` backend) Jobs launched with `qsub` can use directories that Cromwell creates as the working directory of the job, and write files to those directories.
 
-The root directory that Cromwell uses for all workflows (`cromwell-root`) defaults to `./cromwell-executions`.  However, this is can be overwritten with the `-Dbackend.shared-filesystem.root=/your/path` option on the command line.
+The root directory that Cromwell uses for all workflows (`cromwell-root`) defaults to `./cromwell-executions`.  However, this is can be overwritten with the `-Dbackend.shared-filesystem.root=/your/path` option on the command line, or via [Cromwell's configuration file](#configuring-cromwell)
 
-When cromwell runs a workflow through either of these two backends, it first creates a directory `<cromwell-root>/<workflow_uuid>`.  This is called the `workflow_root` and it is the root directory for all activity in this workflow.
+When cromwell runs a workflow, it first creates a directory `<cromwell-root>/<workflow_uuid>`.  This is called the `workflow_root` and it is the root directory for all activity in this workflow.
 
 Each `call` has its own subdirectory located at `<workflow_root>/call-<call_name>`.  This is the `<call_dir>`.  Within this directory are special files written by the backend and they're supposed to be backend specific things though there are commonalities.  For example, having a `stdout` and `stderr` file is common among both backends and they both write a shell script file to the `<call_dir>` as well.  See the descriptions below for details about backend-specific files that are written to these directories.
 
-Any input files to a call need to be localized in the `<call_dir>`.  There are a few localization strategies that Cromwell will try until one works.  Below is the default order specified in `application.conf` but this order can be overridden:
+An example of a workflow output directory would look like this:
+
+```
+cromwell-executions/
+└── three_step
+    └── ec5ae483-bf75-4162-b3ab-a7a52710f3d4
+        ├── call-cgrep
+        │   ├── Users
+        │   │   └── sfrazer
+        │   │       └── projects
+        │   │           └── cromwell
+        │   │               └── cromwell-executions
+        │   │                   └── three_step
+        │   │                       └── ec5ae483-bf75-4162-b3ab-a7a52710f3d4
+        │   │                           └── call-ps
+        │   │                               └── stdout
```

Comment:
I'd suggest adding a brief explanation as to why this hierarchy exists as it's a bit surprising and took me a while to remember why we do this.  :smile: